modules/virt-configuring-vm-with-node-exporter-service.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-configuring-vm-with-node-exporter-service_{context}"]
+= Configuring a virtual machine with the node exporter service
+
+Download the `node-exporter` file on to the virtual machine. Then, create a `systemd` service that runs the `node-exporter` service when the virtual machine boots.
+
+.Prerequisites
+* The pods for the component are running in the `openshift-user-workload-monitoring` project.
+* Grant the `monitoring-edit` role to users who need to monitor this user-defined project.
+
+.Procedure
+
+. Log on to the virtual machine.
+
+. Download the `node-exporter` file on to a virtual machine by using the directory path that applies to the version of `node-exporter` file.
++
+[source,terminal]
+----
+$ wget https://github.com/prometheus/node_exporter/releases/download/v1.3.1/node_exporter-1.3.1.linux-amd64.tar.gz
+----
+
+. Extract the executable and place it in the `/usr/bin` directory.
++
+[source,terminal]
+----
+$ sudo tar xvf node_exporter-1.3.1.linux-amd64.tar.gz \
+    --directory /usr/bin --strip 1 "*/node_exporter"
+----
+
+. Create a `node_exporter.service` file in this directory path: `/etc/systemd/system`. This `systemd` service file runs the `node-exporter` service when the virtual machine is rebooted.
++
+[source,terminal]
+----
+[Unit]
+Description=Prometheus Metrics Exporter
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=1
+User=root
+ExecStart=/usr/bin/node_exporter
+
+[Install]
+WantedBy=multi-user.target
+----
+
+. Enable and start the `systemd` service.
++
+[source,terminal]
+----
+$ sudo systemctl enable metrics
+$ sudo systemctl start metrics
```

Comment:
@sbulage - I think it will be confusing if we change the service name to node_exporter.service. There is value in giving a different service name such as metrics to ensure that the user clearly associates **metrics** as the name given to the service. Also, this would inform the users that they can provide any name of their choice for the service name, it does not have to be node exporter service. I would request @jobbler to comment on my thoughts.

Reply:
Hello @lmandavi check this line, you have mentioned here: "Create a `node_exporter.service`" --> 
https://github.com/openshift/openshift-docs/pull/41475/files#diff-8fe1fa3208b393f512fcc7a66bfca46ae684897182d6d84ec5c7eb1ad127a976R33

And yes, you're right user can provide any name of their choice but in this example it is `node_exporter.sercive` only.

Reply:
@lmandavi When starting and stopping the service it should be node-exporter as @sbulage mentions. This is because line 33 says to create a file node-exporter.service and not metrics.service. The name of the file is where the name of the service comes from. So you could either change the name of the file created to metrics.service or change the sudo systemctl commands from metrics to node-exporter

Reply:
Appreciate the logic and I've made the change.